haas/model.py

```diff
@@ -243,14 +242,14 @@ class Network(Model):
     # The project to which the network belongs, or None if the network was
     # created by the administrator.  This field determines who can delete a
     # network.
-    creator_id = Column(String,ForeignKey('project.id'))
+    creator_id = Column(Integer,ForeignKey('project.id'))
```

Comment:
Rather than specifying the "id" column type in this and the other local calls you changed, let's just remove the type and let SqlAlchemy [infer it from the foreignkey](http://docs.sqlalchemy.org/en/rel_0_9/core/metadata.html#sqlalchemy.schema.Column.params.type_).

This helps us not to repeat ourselves, making it easier if we want to move to a biginteger primary key (#479) or something else.